code/modules/admin/verbs/debug.dm

```diff
@@ -543,31 +543,49 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 
 	for(var/obj/machinery/airalarm/AA in GLOB.machines)
 		var/area/A = get_area(AA)
+		if(!A) //Make sure the target isn't inside an object, which results in runtimes.
+			to_chat(world, "<b>Test Areas:<b> Skipping over [AA] in invalid location, [AA.loc].")
```

Comment:
Instead of doing a world message, could put all of these in a var and then display on a pop up so it's easier to copy/check the output.